source/source_integration_test.go

```diff
@@ -244,13 +244,23 @@ func TestSource_CDC_ReadRecordsInsert(t *testing.T) {
 	_ = source.Teardown(ctx)
 }
 
-func TestSource_CDC_ReadRecordsUpdate(t *testing.T) {
+func TestSource_CDC_UpdateWithVersioning(t *testing.T) {
 	client, cfg := prepareIntegrationTest(t)
 
 	ctx := context.Background()
 	testBucket := cfg[config.ConfigKeyAWSBucket]
 	source := &Source{}
-	err := source.Configure(context.Background(), cfg)
+
+	// make the bucket versioned
+	_, err := client.PutBucketVersioning(ctx, &s3.PutBucketVersioningInput{
```

Comment:
I assume this operation is idempotent, and won't fail if you try enabling versioning for a bucket which already has it enabled?

Reply:
Based on the word "put" I expect it to be idempotent, but even if it's not, we create a new bucket for each test.